### DISCUSSION TITLE
Styles

### DISCUSSION THREAD

**Discussion by Ray7301** [vladmandic/sdnext#2968] [Q&A] [answered]
I come from Automatic 1111, there you could run --styles-file="path to you own style csv's\*.csv to load all your style files.
Now i noticed i could set the path to style files here, but i only see the builtin's. What am i doing wrong. for example i changed the path from models\styles to z:\styles\ but nothing changes in the style selection in the txt2image interface.

**Comment by vladmandic** [comment, marked as answer]
sdnext does not use csv-style styles from a1111. but if you go to settings->paths and set styles path to full pathname of the csv file (not just folder, full path with filename), sdnext will convert it during next startup to its native json format and reset styles path.

**Reply by Ray7301** [reply]
Ah thank you, i abused the style sheets in auto1111 to create a kind of click together prompt system, so have many style sheets with all kinds of sorted lists. I guess that i a little bit harder to do now because every seperate entry will get a json (and i have to set the link to the csv about 119 times haha. But thank you for the information, it does work, setting the csv itself.
